dockers/docker-lldp-sv2/lldpmgrd

```diff
@@ -116,10 +116,13 @@ class LldpManager(object):
             port_table_dict = dict(fvp)
 
             # Get the oper-status for the port
-            port_oper_status = port_table_dict.get("oper_status")
-            log_info("Port name {} oper status: {}".format(port_name, port_oper_status))
-            return port_oper_status == "up"
-
+            if port_table_dict.has_key("oper_status"):
+                port_oper_status = port_table_dict.get("oper_status")
+                log_info("Port name {} oper status: {}".format(port_name, port_oper_status))
+                return port_oper_status == "up"
+            else:
+                log_error("Port name {} does not have key oper_status".format(port_name))
```

Comment:
Let's don't output any error here?

Reply:
For multiple cases we return False. Outputting an error message is the only way to find out the reason for False.

---
In reply to: [304107933](https://github.com/Azure/sonic-buildimage/pull/3152#discussion_r304107933) [](ancestors = 304107933)

Reply:
That key should always be present. I think this log message is helpful for catching unexpected circumstances.

Reply:
The key will not be presented when we creating this table.
So on sonic initialization you can catch some "errors" here.
I think it's not an error at all.

Reply:
Good point. We might see some false alarms at boot time and when reloading config. Maybe it is better to simply remove it.